hack/machineset.sh

```diff
@@ -171,7 +171,7 @@ $(get_spec $infraID $az $provider $byoh)
             publisher: MicrosoftWindowsServer
             resourceID: ""
             sku: $sku
-            version: latest
+            version: "17763.6293.240905"
```

Comment:
Using the latest with Windows Server 2022 seems safe; the fix version should be used with Windows Server 2019 only.